test/teiserver_web/controllers/account/session_controller_test.exs

```diff
@@ -5,6 +5,42 @@ defmodule TeiserverWeb.Account.SessionControllerTest do
   alias Teiserver.Account
   alias Teiserver.Account.Guardian
   alias Teiserver.Config
+  alias Phoenix.ConnTest
+
+  describe "login" do
+    setup do
+      GeneralTestLib.conn_setup(Teiserver.TeiserverTestLib.player_permissions())
+      |> Teiserver.TeiserverTestLib.conn_setup()
+    end
+
+    test "no OTP check when no secret", %{conn: conn, user: user} do
+      conn = GeneralTestLib.login(conn, user.email)
+
+      assert redirected_to(conn) == ~p"/"
+      assert ConnTest.get_flash(conn, :info) == "Welcome back!"
```

Comment:
I wouldn't test that, it brings nothing for this test and is going to get annoying when we make just a copy change. Here you are only concerned about log in logic, not messaging or presentation.